docs/jotai/advanced-recipes/atom-creators.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: Atom creators
+nav: 5.2
+---
+
+## `atomWithToggle`
+
+> `atomWithToggle` creates a new atom with a boolean as initial state & a setter function to toggle it.
+
+This avoids the boilerplate of having to setup another atom just to update the state of the first.
+
+```ts
+import { WritableAtom, atom } from 'jotai'
+import { atomWithStorage } from 'jotai/utils'
```

Comment:
`atomWithStorage` isn't referenced by the custom `atomWithToggle()` immediately below so this import should probably be removed.

Reply:
Yeah, let's remove it. Will you?